OxfordBodleian/Juel-Jensen/BDLaethf28.xml

```diff
@@ -133,13 +174,57 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
 
+                                <desc>At least two hands were involved in the writing of the manuscript, possibly more. 
+                                    The script changes repeatedly throughout, often within the same page (e.g. <locus target="#35 #90 #115"/>), 
+                                    leading to an overall impression of irregularity. All hands belong to the nineteenth–twentieth century.</desc>
+                                <date notBefore="1800" notAfter="1998"/>
```

Comment:
If we do not use the information from the handout, does it make sense to say "1998"? Maybe rather an even number?